src/components/link-button.tsx

```diff
@@ -8,6 +8,20 @@ interface LinkButtonProps {
   app: App;
 }
 
+// Detect the file is opened.
+function findLeafWithFile(app: any, filePath: string): WorkspaceLeaf | null {
```

Comment:
Tip: Locally you can run `npm run lint` to see these errors.

The fix here is to specify the type `App` which is already imported from `obsidian`. For example:

```suggestion
function findLeafWithFile(app: App, filePath: string): WorkspaceLeaf | null {
```